test/etc/nginx/conf.d/test.conf

```diff
@@ -165,6 +165,15 @@ BwIDAQAB
         try_files index.html =404;
     }
 
+    location /secure/auth-query-string/hs256 {
+        auth_jwt_enabled on;
+        auth_jwt_location QUERY_STRING=jwt-token;
+        auth_jwt_algorithm HS256;
+
+        alias /usr/share/nginx/html/;
+        try_files index.html =404;
+    }
+
```

Comment:
```suggestion
    location /secure/auth-querystring/hs256 {
        auth_jwt_enabled on;
        auth_jwt_location QUERYSTRING=jwt-token;
        auth_jwt_algorithm HS256;

        alias /usr/share/nginx/html/;
        try_files index.html =404;
    }

```

Reply:
Thanks, fixed in commit: https://github.com/TeslaGov/ngx-http-auth-jwt-module/pull/107/commits/7dd8b005b8e932175e7cef4f34c026eb1b9c6fdf
Also changed the querystring to token.